index.js

```diff
@@ -17,3 +78,89 @@ for (let i = 0; i < menuItems.length; i += 1) {
     overlay.style.display = 'none';
   });
 }
+
+closeModalBtn.addEventListener('click', () => {
+  modal.style.display = 'none';
+});
+
+// Loop through dummy data to make web page page dynamic
+
+for (let i = 0; i < projects.length; i += 1) {
+  // HTML Template for card in portfolio section
+  const cardSection = `
+  <div class="card grid">
+    <div>
+      <img src="${projects[i].imageUrl}" class="image-auto-h" alt="Tonic" />
+    </div>
+    <div class="change-row">
+    <h2 class="m0 font-40 h-0 f-color">${projects[i].title}</h2>
+    <ul class="d-flex space-x-24 p-highlight mb-32 project-summary">
+    </ul>
+    <p>
+      ${projects[i].description}
+    </p>
+    <ul class="d-flex space-x-1 list-tags list-style-n no-pad flex-wrap stack-list">
+    </ul>
+    <button class="btn-action mt-40 view-details" type="button">See project</button>
+  </div>
+  </div>
```

Comment:
- [ ] Kindly make sure to add the horizontal line that separates the project stack and the two buttons.
- [ ] KIndly make the two buttons aligned correctly and if you see carefully, they touch both ends of their container. Kindly fix that.

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/225888544-b2f52a11-9e0a-4bdf-8f62-4990935b2d2d.png)

   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/225888629-9d55a4e5-abe2-45a8-9913-187c6e57c272.png)